build.sbt

```diff
@@ -1,8 +1,9 @@
 import sbt.Keys._
 
-ThisBuild / scalaVersion := "2.12.13"
+ThisBuild / scalaVersion := Version.scala211
```

Comment:
I'd like to see `Version.scala212` as the default here. :D 
2.11 is not a priority atm, more like a legacy crossompilation that I'd really like to drop. 
We use GT mostly with 2.12.

Reply:
sorry, that must have been a typo when I originally made this change -- I've been cross-compiling most of the time and didn't notice.